internal/init/prompts.go

```diff
@@ -88,6 +88,36 @@ func ValidateSAK(input string) error {
 	return nil
 }
 
+// ValidateProjectName validates Project Name field user input.
+func ValidateProjectName(input string) error {
+	// the first 62 char out of base64 and -
+	var pName = regexp.MustCompile(`^[A-Za-z0-9-]{1,15}$`)
+	if !pName.MatchString(input) {
+		return errors.New("Invalid project-name (cannot contain special chars except '-' & max len of 15)")
+	}
+	return nil
+}
+
+func validateRootDomain(input string) error {
+	// FIXME: this regex would still accept subdomains a.co.uk a permanent solution matching all TLDs
```

Comment:
Let's just make it more permissive and allow TLDs with dots like your example of `a.co.uk`, but make sure that we have enough of a description during the prompt to give them an idea of why they need to enter a root domain and leave it up to them.